front/pages/api/apps/[user]/[sId]/runs/index.ts

```diff
@@ -108,8 +122,11 @@ export default async function handler(req, res) {
           });
 
           try {
-            for await (const chunk of streamRes.body) {
+            for await (const chunk of streamChunks(streamRes.body)) {
               res.write(chunk);
+              // TODO: figure out if .flush() does anything
+              // otherwise figure out why typescript complains
+              // @ts-ignore
```

Comment:
Did you figure that out?

Reply:
I could not find any flush() method on the ts files.

Reply:
It seems to work the same without from my tests

Reply:
I removed the `.flush`